python_translators/translation_response.py

```diff
@@ -0,0 +1,7 @@
+from python_translators.translation_costs import TranslationCosts
+
+
+class TranslationResponse(object):
+    def __init__(self, translations: [str], costs: TranslationCosts):
```

Comment:
If you look at my implementation of the BestEffortTranslator you'll see that it returned a confidence together with every translation. 

I personally like that idea. 

For example, it seems that Glosbe returns the results in some kind of order of probability... thus, it would make sense to have a decreasing order of confidence. Also a translation uploaded by the professor of the class, would have higher confidence than the Google one. 

Do you think that it would make sense to add confidence to the TranslationResponse?